### PR TITLE
Frocksdb release for 5.17.2-artisans-2.0

### PR DESCRIPTION
**Pull request by Myasuka**
Add Java API changes to `HISTORY.md`.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # FRocksdb Change Log
 
-# 5.17.2-artisans (07/02/2019)
+# 5.17.2-artisans-1.1 (11/26/2019)
```
Comment (azagrebin) [inline HISTORY.md]: it seems we are adding some changes which are unrelated to the changes of the previous version `1.0`.
should we rather release this version as `2.0`?

File: HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+### Java API Changes
+* Allow rocks java to explicitly create `WriteBufferManager`.
+* Punch through more flags for rocks java's `BlockBasedTableConfig`.
```
Comment (azagrebin) [inline HISTORY.md]: I would write it as:
`Add more flags for BlockBasedTableConfig, mostly around caching index + filter blocks and partitioned filters`
It would also nice to briefly mention for what we need these changes in Flink

**Review by azagrebin — COMMENTED** [review]
Thanks for this PR @Myasuka 
I have left couple of comments to address before merging

**Comment by Myasuka** [comment]
@azagrebin Thanks for your review, already updated this PR.

**Review by azagrebin — APPROVED** [review]
Thanks for addressing the comments @Myasuka 
LGTM, merging this
